pkg/loader/openapi.go

```diff
@@ -80,10 +92,9 @@ func getOpenAPITools(t *openapi3.T) ([]types.Tool, error) {
 				},
 			}
 
-			toolNames = append(toolNames, tool.Parameters.Name)
-
-			// Handle query, path, and header parameters
-			for _, param := range operation.Parameters {
+			// Handle query, path, and header parameters, based on the parameters for this operation
+			// and the parameters for this path.
+			for _, param := range append(operation.Parameters, pathObj.Parameters...) {
```

Comment:
Parameters can also be defined at the path level (`pathObj.Parameters`), so we are now handling those as well.